learn/cmd/levy/main.go

```diff
@@ -91,12 +91,13 @@ type unsealCmd struct {
 }
 
 func (c *exportCmd) Run() error {
-	if c.ExportType == "answerkey" && c.WithAnswersMarked {
+	if (c.ExportType == "answerkey" || c.ExportType == "catalog") && c.WithAnswersMarked {
```

Comment:
I would change this condition to match the error message now that there are equal number of options either way:

    if c.WithAnswersMarked && c.ExportType != "all" && c.ExportType != "html"

Reply:
yeah, agreed. More readable too. thank you fixed.